byoqm/source_coordinator/query_translations.py

```diff
@@ -113,19 +119,21 @@
         "function_block": "(method_declaration body: (block) @function.block) (constructor_declaration body: (constructor_body) @function.block)",
         "return": "(return_statement)",
         "nested_controlflow1": """
-                                (_
+                                (method_declaration
                                     body: (block [
                                     (if_statement
-                                        consequence: (block) @cons)
+                                        consequence: (_) @cons)
                                     (if_statement
-                                        consequence: (block) @cons
+                                        consequence: (_) @cons
                                         alternative: [
                                             (_ [alternative: (_) consequence: (_) ] @cons)
                                             (block) @cons])
                                     (for_statement
                                         body: (block) @cons)
                                     (while_statement
-                                        (block) @cons)]))
+                                        (block) @cons)
+                                    (switch_expression
+                                        body: (switch_block (switch_block_statement_group) @cons))]))
                                 """,
         "nested_controlflow2": """
```

Comment:
As far as I understand, `nested_controlflow1` and `nested_controlflow2` are a "query starter" and "query _continuer_", maybe some better names would be:

- `nested_controlflow_kickoff`
- `nested_controlflow`

Reply:
One of the queries is for all of the top-level nodes and the other is for all subsequent nodes

Reply:
Yea, I just want the names to reflect that instead of `1` and `2` :)